pytorch_lightning/callbacks/model_checkpoint.py

```diff
@@ -45,7 +45,15 @@
 warning_cache = WarningCache()
 
 
-class ModelCheckpoint(Callback):
+class Checkpoint(Callback):
+    r"""
+    This is the base class for Model checkpointing. Expert users may want to subclass it in case of writing
+    custom :class:`~pytorch_lightning.callbacks.model_checkpoint.ModelCheckpoint` callback, so that
+    the trainer recognizes the custom class as a checkpointing callback.
+    """
+
+
+class ModelCheckpoint(Checkpoint):
```

Comment:
`_FaultToleranceCheckpoint` should also be updated to subclass it

Reply:
It is now